tests/InputFiles/RadiationTransport/M1Grey/ConstantM1.yaml

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
Presumably the previous commit doesn't work without these changes, so squash these two commits together.